examples/cryptocurrency-advanced/backend/src/transactions.rs

```diff
@@ -44,7 +45,7 @@ pub enum Error {
 }
 
 /// Transfer `amount` of the currency from one wallet to another.
-#[derive(Clone, Debug, ProtobufConvert)]
+#[derive(Clone, Debug, ProtobufConvert, BinaryValue, ObjectHash)]
 #[exonum(pb = "proto::Transfer", serde_pb_convert)]
```

Comment:
I think we should avoid of using `exonum` for attribute name here, and also is there any need to have `serde_pb_convert` attribute that looks very strange and unclear.